crates/biome_js_analyze/src/lint/nursery/use_single_js_doc_asterisk.rs

```diff
@@ -0,0 +1,305 @@
+use biome_analyze::{
+    Ast, FixKind, Rule, RuleDiagnostic, RuleSource, context::RuleContext, declare_lint_rule,
+};
+use biome_console::markup;
+use biome_js_syntax::{JsModule, JsSyntaxToken};
+use biome_module_graph::JsdocComment;
```

Comment:
I feel bit weird using function from module graph, despite not using module graph itself. Should it be moved somewhere else?

Reply:
Heh, sorry about that! I knew this moment would come 😅

I just didn’t know a better place to move it to and creating a whole crate for just the one struct felt like overkill. But yeah, it’s starting to make more sense now. @ematipico Would you know a better place for it, or shall we create a `biome_jsdoc` crate?

Reply:
A new crate is an option. I can propose two other options without creating a new crate:
- having a `comments_ext.rs` inside the `biome_js_syntax` crate
- make jsdocs part of the semantic model crate (OXC does this)

Unfortunately I don't how the JsDoc structs work under the hood, so take my suggestions with a grain of salt

Reply:
I think I'd lean towards a new crate still... @minht11 you can do it in this PR, but feel free to leave it, and I'll do it as a follow-up later.

Reply:
Added new crate